samples/word/99-preview-apis/content-control-ondeleted-event.yaml

```
@@ -0,0 +1,161 @@
+id: word-content-control-ondeleted-event
+name: Content control onDeleted event
+description: Register and trigger onDeleted event on content controls.
```

Consider framing the name and description more around the scenario of deleting the controls.
```suggestion
name: Delete content controls
description: Register and trigger onDeleted event to track the removal of content controls.
```